packages/react-devtools-shared/src/devtools/views/ErrorBoundary/githubAPI.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import {Octokit} from '@octokit/rest';
+
+export type GitHubIssue = {|
+  title: string,
+  url: string,
+|};
+
+export async function searchGitHubIssues(
+  message: string,
+): Promise<GitHubIssue | null> {
+  // Remove Fiber IDs from error message (as those will be unique).
+  message = message.replace(/"[0-9]+"/, '');
+
+  const filters = [
+    // Unfortunately "repo" and "org" filters don't work
+    // Hopefully the label filter will be sufficient.
+    'in:title',
+    'is:issue',
+    'is:open',
+    'is:public',
+    'label:"Component: Developer Tools"',
+  ];
```

Comment:
Are you sure you're using the filters correctly? This works A-OK:

https://api.github.com/search/issues?q=repo:facebook/react+is:issue+is:open+is:public+in:title+Error

Reply:
> Are you sure you're using the filters correctly?

Yes, it works when used in the URL but it doesn't work when passed to the API. When I add either `repo` or `org` filters to the query it fails with:
> The listed users and repositories cannot be searched either because the resources do not exist or you do not have permission to view them.

Please let me know if you can figure out a way to make this work. I'd be happy to tweak the search.

Reply:
I just ran this through Octokit and it provided the right results:

```js
await octokit.rest.search.issuesAndPullRequests({ q: 'Error repo:facebook/react is:issue is:open' })

{
    "total_count": 191,
    "incomplete_results": false,
    "items": Array(30)
}

await octokit.rest.search.issuesAndPullRequests({ q: 'Error org:facebook is:issue is:open' })

{
    "total_count": 5269,
    "incomplete_results": false,
    "items": Array(30)
}
```

Reply:
That's so bizarre. Literally what I tried doing yesterday and it wasn't working.

Reply:
#21292